Documentation/Power.md

```diff
@@ -37,7 +37,7 @@ Therefore the Pinecil and TS100 perform poorly when run off 12V power supplies a
 
 These soldering irons use a FET to switch the power to the soldering iron tip. This is a P-MOSFET and its controlled via a small transistor circuit, which in turn is controlled via the MCU (i.e., STM32). The MCU controls this PWM output proportional to the output from the PID control loop running in the software.
 
-To measure the tip temperature in the iron, the iron has a small op-amp connected across the terminals at the cold end of the tip. This is setup to measure the voltage across the same terminals that are used to power the tip. In order to read the very small voltage generated by the [thermocouple cold junction](/Documentation/Temperature.md), the irons output must be turned off for a moment.
+To measure the tip temperature in the iron, the iron has a small op-amp connected across the terminals at the cold end of the tip. This is setup to measure the voltage across the same terminals that are used to power the tip. In order to read the very small voltage generated by the [thermocouple cold junction](/Documentation/Temperature.md), the iron output must be turned off for a moment.
```

Comment:
This is wrong!
It has to be either how it was before : *'the irons output'* or *'the output of the iron'*.